### DISCUSSION TITLE
Is it possible to including portions of numbered lists with numbering kept in sequence?

### DISCUSSION THREAD

**Discussion by bimCelum** [mondeja/mkdocs-include-markdown-plugin#199] [Q&A] [answered]
Hi,
we are using mkdocs-include-markdown-plugin with Material MKDocs, and it works really well.

There is just one case I'm having trouble with: We have several step-by-step numbered lists which are partially repeated in various places. So we want to use an include statement to include the repeating steps in the places from the same include-file. The content gets included just fine, but the numbering of the included steps restarts at 1 and the steps that come after the include statement (but are part of the same list) also restart at 1 instead of continuing.

So my question is - is this possible somehow? What would the syntax or config for that be?

Here are some screenshots for context:

Include-statement in .md file:
![image](https://github.com/mondeja/mkdocs-include-markdown-plugin/assets/20797331/988c381a-e702-46bc-8b09-adb88672ff37)

Final list in HTML output:
![image](https://github.com/mondeja/mkdocs-include-markdown-plugin/assets/20797331/4ae8fa7d-9ae1-46ed-9489-2057a620fe55)

Any help will be much appreciated.

Birgit.

**Comment by mondeja** [comment, marked as answer]
Several things probably happening here that you must take into account:

- You can use ordered lists with all ones as numbers and will be rendered correctly, see https://www.markdownguide.org/basic-syntax/#ordered-lists
- If you don't use [`comments=false`](https://github.com/mondeja/mkdocs-include-markdown-plugin?tab=readme-ov-file#include-markdown_comments), two newlines will be included in the content, probably resetting the count. `comments=false` will be the default from v7.0.0, see https://github.com/mondeja/mkdocs-include-markdown-plugin/issues/194
- Another option is to only include the content of the list item and not the list item itself (also with `comments=false`) with something like `1. {% include-markdown ... %}`.
